### PR TITLE
Success

### PR DESCRIPTION
**Pull request by Andrewtxt2**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -3,8 +3,20 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
```
Comment (Ivan-Shvorak) [inline src/getHumanAge.test.js:6]: Please restore this test. We need it

File: src/getHumanAge.test.js [hunk 2]
```diff
@@ -3,8 +3,20 @@
...
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+  test('should return an array with two elements', () => {
```
Comment (Ivan-Shvorak) [inline src/getHumanAge.test.js:6]: We are checking it in each case, no need to check separately that array should return an array with 2 elements

File: src/getHumanAge.test.js [hunk 3]
```diff
@@ -3,8 +3,20 @@
...
+    expect(result).toBeInstanceOf(Array);
+    expect(result.length).toEqual(2);
+  });
+
+  test('should return [0,0] for catAge and dogAge < 1', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+    expect(getHumanAge(0.5, 0.2)).toEqual([0, 0]);
+  });
+
+  test('should return correct human ages for catAge and dogAge', () => {
```
Comment (Ivan-Shvorak) [inline src/getHumanAge.test.js:18]: More tests are required. Please use Equivalence classes and Boundary values to check all main cases

**Review by Ivan-Shvorak — CHANGES_REQUESTED** [review]
Please check the comments
